react/features/base/conference/functions.js

```diff
@@ -180,9 +180,9 @@ export function getConferenceName(stateful: Function | Object): string {
     const state = toState(stateful);
     const { callee } = state['features/base/jwt'];
     const { callDisplayName } = state['features/base/config'];
-    const { pendingSubjectChange, room, subject } = getConferenceState(state);
+    const { localSubject, room, subject } = getConferenceState(state);
```

Comment:
shouldn't we still use `pendingSubjectChange`?

Reply:
`pendingSubjectChange` is only used to update the global `subject` now (when the participant has a moderator role) and for local purposes we provide this new command instead.